src/org/omegat/core/team2/impl/GITRemoteRepository2.java

```diff
@@ -191,8 +192,7 @@ public String getFileVersion(String file) throws IOException {
 
     protected String getCurrentVersion() throws IOException {
         try (RevWalk walk = new RevWalk(repository)) {
-            Ref localBranch = repository.findRef("HEAD");
-            RevCommit headCommit = walk.lookupCommit(localBranch.getObjectId());
+            RevCommit headCommit = walk.lookupCommit(repository.resolve("HEAD"));
```

Comment:
Use [`resolve("HEAD")`](https://download.eclipse.org/jgit/site/5.12.0.202106070339-r/apidocs/org/eclipse/jgit/lib/Repository.html#resolve-java.lang.String-) instead of `findRef("HEAD").getObjectId()`
because "HEAD" is a primary supported string with `resolve`